internal/common/common.go

```diff
@@ -261,3 +300,49 @@ func ValidateObjectKey(key ctrlclient.ObjectKey) error {
 	}
 	return nil
 }
+
+// SyncableSecretMetaData provides common data structure that extracts the bits pertinent
```

Comment:
Moved this from helpers because of circular dependences between `helpers` and `common`

Reply:
:+1: